tests/integration/transforms/transforms_test.go

```diff
@@ -93,6 +94,9 @@ func Validator(t *testing.T, stack integration.RuntimeValidationStackInfo) {
 			// we change the provider but because this is a remote component resource it ends up empty in state.
 			assert.Equal(t, "", res.Provider)
 		}
+		// if res.URN.Name() == "res8" {
+		// 	foundRes8 = true
+		// }
```

Comment:
Assuming it's fine for us to leave these in right now, as opposed to just adding them in a later PR?

Reply:
I was leaving it in for convenience, but also fine with removing it if that's preferred.